kibana-reports/public/components/report_definitions/create/create_report_definition.tsx

```diff
@@ -35,6 +35,8 @@ interface reportParamsType {
   report_name: string;
   report_source: string;
   description: string;
+  header: string;
+  footer: string;
```

Comment:
This is only for create report definition, also need to add support for edit report definition

Reply:
Good catch, will add